eng/testing/xunit.runner.json

```diff
@@ -1,4 +1,5 @@
 {
   "$schema": "https://xunit.net/schema/current/xunit.runner.schema.json",
-  "longRunningTestSeconds": 120
+  "longRunningTestSeconds": 120,
+  "parallelizeAssembly": false
```

Comment:
Was this to fix the AppHost issue?

Reply:
If so, is it not possible to only remove parallelization of tests within that assembly instead?

Reply:
This was to fix the test failures due to hitting the io handle limit on Linux.

Reply:
We should set this only for `Aspire.Hosting`, so the other assemblies can still run in parallel. I'm assuming that we start off all the test assemblies in parallel and xunit decides how many to actually run concurrently. And this would be the case where we have a single step of `run non-helix steps`.

Reply:
Thing is, this [apparently defaults to `false`](https://xunit.net/docs/configuration-files#parallelizeAssembly), so I'm not actually sure if this is what fixed it. I didn't realize the file was shared when I made the change (it's linked into the `Aspire.Hosting.Tests` project).

Reply:
Happy to change it so that `Aspire.Hosting.Tests` has its own copy of this file and just have the change there.

Reply:
> defaults to false

nvm then. we can drop the change then, but if you want to keep it then separate copy would be great, IMHO.